src/main/java/org/broadinstitute/hellbender/cmdline/programgroups/BwaMemUtilitiesProgramGroup.java

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.hellbender.cmdline.programgroups;
+
+import org.broadinstitute.barclay.argparser.CommandLineProgramGroup;
+import org.broadinstitute.hellbender.utils.help.HelpConstants;
+
+/**
+ * Program group for using non-essential part of JNI binding to Bwa Mem
```

Comment:
My impression is that the binding is for alignment only, and the distribution of images of those indices is more desirable in the Spark context. I can certainly remove that if you feel that would be more appropriate.